platform/positionmanager/manifest.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: position-manager-service
+spec:
+  template:
+    spec:
+      containers:
+        - name: position-manager
+          image: pocketsizefund/position-manager:20240903.2
+          env:
+            - name: DARQUBE_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: DARQUBE_API_KEY
+            - name: ALPACA_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: ALPACA_API_KEY
+            - name: ALPACA_API_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: ALPACA_API_SECRET
+            - name: ALPHA_VANTAGE_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: ALPHA_VANTAGE_API_KEY
+            - name: EDGAR_USER_AGENT
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: EDGAR_USER_AGENT
+---
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: position-manager-trigger
+spec:
+  schedule: "42 12 * * *"
```

Comment:
Why 42?

Reply:
was just testing and that was a close time, removing the whole trigger anyway